tm2/pkg/db/common_test.go

```diff
@@ -8,8 +8,8 @@ import (
 
 	"github.com/gnolang/gno/tm2/pkg/db"
 	"github.com/gnolang/gno/tm2/pkg/db/internal"
-	"github.com/jaekwon/testify/assert"
-	"github.com/jaekwon/testify/require"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
didn't swapped `actual` and `expected` in this file as they were incorrect before.

same goes with some other test files.